source/manuals/aws-tagging.html.md.erb

```diff
@@ -36,13 +36,13 @@ Currently, we do not enforce tags.
 
 In future, we may wish to consider mechanisms such as alerting on untagged resources, or automatically deleting untagged resources.
 
-## Tags used in GOV.UK Sign In
+## Tags used in GOV.UK One Login
 
-GOV.UK Sign In is using the following tags:
+GOV.UK One Login is using the following tags:
 
 ### Mandatory
 
- - `Product`: should be `GOV.UK Sign In`
+ - `Product`: should be `GOV.UK One Login`
```

Comment:
@paroxp, could I confirm if we merged that, that I'm not going to break any reports we have.

I think now because the tags I've already seen are a bit of a mess of different names already?

And I know we're about to transition to our own org too?

Reply:
If we were to break any reports, which there may be little of, we shouldn't be blocked on iterating on our changes :) 

I think we can take the heat, and fix/adjust the reports later if that turns to be an issue.

Reply:
@deanwilson gonna treat that as a "happy to go" from my program

Reply:
@paroxp would you mind giving this a ✅ ?